test/components/plots/plotTests.ts

```diff
@@ -36,21 +36,21 @@ describe("Plots", () => {
 
       assert.strictEqual(1, r.renders, "initial render due to addDataset");
 
-      dFoo.broadcaster.broadcast();
+      (<any>dFoo)._dispatchChange();
```

Comment:
Since `_dispatchChange()` isn't public, the better way to trigger an update might be something like
```Typescript
dFoo.data(dFoo.data());
```